Core v2 docs update (#819)
spatialos/gdk-for-unity: docs/content/ecs/component-updates.md

}
```

When a SpatialOS entity is [checked out]({{urlRoot}}/content/glossary#checking-out), its components are added to the corresponding ECS entity as part of the entity's check out process.

### How to send a component update

> To send a component update, set the component to the value to be sent. A component update is constructed and sent at the end of the current update loop.

Reviewer: ```suggestion
To send a component update, change the value of at least one field of the component . A component update is constructed and sent at the end of the current update loop.
```

still a bit weird, but maybe a bit clearer?

Author: Agreed

}
```

#### Component update edge case

> There is one edge case that you need to be careful about which is a result of how component changes are tracked and [C# reference type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/reference-types) semantics.

Reviewer 2: Breaking up a long sentence with a crafty colon.
```suggestion
There is one edge case that you need to be careful about: this is a result of how component changes are tracked and [C# reference type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/reference-types) semantics.
```

> Simply adding or removing elements to a `List<T>` or a `Dictionary<K, V>` field in a component is not sufficient to trigger a component update. Instead, you need to set the field back to itself to trigger the update. This applies to `List<T>` and `Dictionary<K, V>` fields within types as well.

Reviewer 2: I commented suggesting for an example....before I spotted the excellent example - maybe call out the example?

```suggestion
Simply adding or removing elements to a `List<T>` or a `Dictionary<K, V>` field in a component is not sufficient to trigger a component update. Instead, you need to set the field back to itself to trigger the update, as shown in the example below. This applies to `List<T>` and `Dictionary<K, V>` fields within types as well.
```
